fe/src/main/jade/backdoor.jade

```diff
@@ -0,0 +1,14 @@
+html
+  head
+    script(src="http://code.jquery.com/jquery-1.10.1.min.js")
+    script(src="/js/backdoor.js")
+  body
```

Comment:
This used to be templated in server.rb, I have moved it into the front end. Some js will fill out the correct action.